common/djangoapps/third_party_auth/settings.py

```diff
@@ -84,3 +85,15 @@ def apply_settings(django_settings):
         'social.apps.django_app.context_processors.backends',
         'social.apps.django_app.context_processors.login_redirect',
     )
+
+    set_strategy_getter()
+
+
+def set_strategy_getter():
+    """
+    Set strategy loader method to workaround current strategy getter needed on
+    get_user() method on authentication backends.
+    """
+    from social.strategies.utils import set_current_strategy_getter
+    from social.apps.django_app.utils import load_strategy
+    set_current_strategy_getter(load_strategy)
```

Comment:
@doctoryes, @cpennington Please take a look at this.

**Background**
```
Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 108, in get_response
    response = middleware_method(request)
  File "/edx/app/edxapp/edx-platform/common/djangoapps/student/middleware.py", line 19, in process_request
    user_account = UserStanding.objects.get(user=user.id)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/utils/functional.py", line 225, in inner
    self._setup()
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/utils/functional.py", line 365, in _setup
    self._wrapped = self._setupfunc()
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/contrib/auth/middleware.py", line 22, in <lambda>
    request.user = SimpleLazyObject(lambda: get_user(request))
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/contrib/auth/middleware.py", line 10, in get_user
    request._cached_user = auth.get_user(request)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/contrib/auth/__init__.py", line 174, in get_user
    user = backend.get_user(user_id)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/social/backends/base.py", line 188, in get_user
    return strategy.get_user(user_id)
AttributeError: 'NoneType' object has no attribute 'get_user'
```
I was getting this issue after django1.8 upgrade, when I logged into LMS using social auth and go to studio then this error was occurring. 

And I think reason of this issue is here in [python-social-auth] (https://github.com/omab/python-social-auth/blob/master/social/apps/django_app/__init__.py#L16-L21).

So to fix this I added the above function, what do you guys think about this?